src/terra-dev-site/guide/Internationalization/InternationalizationIntro.a.guide.md

```diff
@@ -0,0 +1,85 @@
+# Internationalization (i18n) Intro
+
+Terra UI provides builtin translations and LTR / RTL support which makes Terra UI powered apps ready to use in several countries with ease. All apps built with Terra UI are required to set up translations, even if they only intend to support English. As we support apps built for use around the world, we want apps to be prepared for internationalization from the beginning of development. It is easier to add additional locales and translations to an app built with internationalization and translations from the beginning compared to retrofitting internalization and translations into an app as it evolves into new markets.
+
+# Getting Started
+Install [terra-base](https://www.npmjs.com/package/terra-base) and [terra-toolkit](https://www.npmjs.com/package/terra-toolkit).
+
+If you're using npm, run the following command:
+
+```
+npm install terra-base terra-toolkit
+```
+
+If you're using yarn, run the following command:
+
+```
+yarn add terra-base terra-toolkit
+```
+
+# Terra Base
+The first part of setting up internalization and translations in your app requires installing and configuring terra-base. This component is designed to manage loading locale information such as translations and locale data, handling locale switching and loading sate, and combining customized translation messages from an application with Terra UI component translations.
```

Comment:
locale switching and loading ~sate~ **state**,

Reply:
Should we specify the site directionality will be loaded with the locale info?

Reply:
We should mention 'en' is the fallback locale for terra-Base and all apps much included 'en' translations at a minimum

Reply:
Updated in this commit: https://github.com/cerner/terra-ui/pull/132/commits/cc0c8713331225234ca6d889d6cca8802fbc4d3c